spec/locales/en/translation.json

```diff
@@ -50,6 +50,7 @@
     "load": "Load",
     "logo": "Logo",
     "metadataTooltip": "View information/metadata about this object",
+    "mirrorTooltip": "Mirror image",
```

Comment:
This has nothing to do with your PR, but is there a way to reuse the translation files in spec, rather than having to edit two files every time there is a change?